src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -31,19 +32,16 @@ export class GestureHandlerWebDelegate
       );
     }
 
+    this.ready = true;
+
     this.gestureHandler = handler;
     this.view = findNodeHandle(viewRef) as unknown as HTMLElement;
 
     const config = handler.getConfig();
 
-    this.addContextMenuListeners(config);
-
-    this.view.style['userSelect'] = config.userSelect ?? 'none';
-    this.view.style['webkitUserSelect'] = config.userSelect ?? 'none';
-
-    this.view.style['touchAction'] = config.touchAction ?? 'none';
-    // @ts-ignore This one disables default events on Safari
-    this.view.style['WebkitTouchCallout'] = 'none';
+    this.setUserSelect(config.enabled!); // `enabled` has default value of true if it was not provided
```

Comment:
In that case, can the type be changed so it's always defined instead of non-null asserts?